src/app.rs

```diff
@@ -50,4 +50,5 @@ pub fn rocket() -> Rocket<Build> {
 				artifacts::create,
 			],
 		)
+		.mount("/static", FileServer::from("static"))
```

Comment:
Will this work properly in prod? I.e. will `/static` be available?

Reply:
Needs to be added to the Dockerfile like templates: https://github.com/beyondessential/tamanu-meta-server/blob/main/Dockerfile#L56

Reply:
Ah, makes sense! Thank you